src/component/layout/Header/Header.js

```diff
@@ -1,58 +1,102 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
+import Container from 'react-bootstrap/Container';
+import Nav from 'react-bootstrap/Nav';
+import Navbar from 'react-bootstrap/Navbar';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import { TbSearch } from 'react-icons/tb';
 import { TbCat } from 'react-icons/tb';
 import { TbDog } from 'react-icons/tb';
-import { TbSearch } from 'react-icons/tb';
-// import { TbHeart } from 'react-icons/tb';
 import style from './Header.module.css';
 
-const Header = () => {
+function Header() {
```

Comment:
If you decided to go with function declaration rather than function expression, try to follow the same pattern across the whole app.

Reply:
I will make sure that everything is consistent during the resolution of the merch conflict.